robot/resources/lib/python_keywords/http_gate.py

```diff
@@ -161,22 +162,44 @@ def upload_via_http_gate(cid: str, path: str, endpoint: str, headers: dict = Non
 
 @allure.step("Upload via HTTP Gate using Curl")
 def upload_via_http_gate_curl(
-    cid: str, filepath: str, endpoint: str, large_object=False, headers: dict = None
+    cid: str,
+    filepath: str,
+    endpoint: str,
+    large_object=False,
```

Comment:
Manual control of this flag is error prune.
Can we do it other way so that the code will decide it?